### ISSUE TITLE
Adding Continue with Guest Option

### ISSUE THREAD

**Issue by iamsohelx** [notsoocool/codecache#46]
**Is your feature request related to a problem? Please describe.**
Hey @notsoocool when someone visit your website and if he/she dont want to give any details, in that case the user can still visit your website using the continue with guest option.
**Describe the solution you'd like**
in you website we can add a continue with guest option [ Frontend + Backend ]
So that user can still visit your website without logging in,
**It Will improve the User Experience**
**Describe alternatives you've considered**

**Additional context**

**Comment by iamsohelx** [comment]
@notsoocool Could you please assign this task to me.

**Comment by menerucha** [comment]
@notsoocool Could you please assign me this task?

**Comment by notsoocool** [comment]
Hey @iamsohelx, you can go ahead with the feature, but please keep the following things in mind:

1. The sign-in/sign-out should be triggered when trying to access a separate snippet page.
2. When clicking the "Add Snippet" button, the same trigger should be activated.
3. For guests, there should be a unique ID assigned to them, so they can be identified separately even without logging in.

Make sure these points are considered during development. Let me know if you have any questions!

**Comment by notsoocool** [comment]
Hey @menerucha, you can also go ahead with the issue, but please keep the points mentioned above in mind. Also, go through the code first to get a basic idea before starting.

**Comment by iamsohelx** [comment]
Hii @notsoocool please assign the task to only one person at a time, I have raised the issue please assign it to me only 🙏🏻.

**Comment by iamsohelx** [comment]
> Hey @iamsohelx, you can go ahead with the feature, but please keep the following things in mind:
> 
> 1. The sign-in/sign-out should be triggered when trying to access a separate snippet page.
> 2. When clicking the "Add Snippet" button, the same trigger should be activated.
> 3. For guests, there should be a unique ID assigned to them, so they can be identified separately even without logging in.
> 
> Make sure these points are considered during development. Let me know if you have any questions!

Thanks brother, i will make sure i didn't do any mistake

**Comment by notsoocool** [comment]
Hi @iamsohelx ,

It’s been over a week without any updates. I will be closing this issue by the end of the day. If you have any progress to share, please do so before then.

Thank you!

**Comment by menerucha** [comment]
I am still working on this issue

**Comment by notsoocool** [comment]
@menerucha you are already working on another issue.
